Sources/MessagingPush/MessagingPush.swift

```diff
@@ -10,6 +10,9 @@ public class MessagingPush: ModuleTopLevelObject<MessagingPushInstance>, Messagi
     @Atomic public private(set) static var shared = MessagingPush()
     private var globalDataStore: GlobalDataStore
 
+    // singleton instance of module configuration
+    @Atomic public static var moduleConfig: MessagingPushConfigOptions = .init()
```

Comment:
This is meant to be an in-memory store for the MessagingPush config options. The DiGraph references this value to provide to dependencies when they need it.